themes/default/content/blog/the-industry-comes-together-for-the-cloud-engineering-summit-2021/index.md

```diff
@@ -0,0 +1,93 @@
+---
+title: "The Industry Comes Together for the Cloud Engineering Summit"
+
+# The date represents the post's publish date, and by default corresponds with
+# the date this file was generated. Posts with future dates are visible in development,
+# but excluded from production builds. Use the time and timezone-offset portions of
+# of this value to schedule posts for publishing later.
+date: 2021-10-06T06:00:00-07:00
+
+# Draft posts are visible in development, but excluded from production builds.
+# Set this property to `false` before submitting your post for review.
+draft: false
+
+# Use the meta_desc property to provide a brief summary (one or two sentences)
+# of the content of the post, which is useful for targeting search results or social-media
+# previews. This field is required or the build will fail the linter test.
+meta_desc: |
+    We are thrilled to have an amazing group of sponsors participating in the 2021 Cloud Engineering Summit, helping us pull off this amazing event.
+
+# The meta_image appears in social-media previews and on the blog home page.
+# A placeholder image representing the recommended format, dimensions and aspect
+# ratio has been provided for you.
+meta_image: meta.png
+
+# At least one author is required. The values in this list correspond with the `id`
+# properties of the team member files at /data/team/team. Create a file for yourself
+# if you don't already have one.
+authors:
+    - isaac-harris
+
+# At least one tag is required. Lowercase, hyphen-delimited is recommended.
+tags:
+    - cloud-engineering-summit
+
+# See the blogging docs at https://github.com/pulumi/docs/blob/master/BLOGGING.md.
+# for additional details, and please remove these comments before submitting for review.
+---
+
+When we launched the first-ever Cloud Engineering Summit last year, we were humbled by the generosity and support from partners and sponsors across the industry who helped to make the summit a success. For this year’s event, we’re equally thrilled to have an amazing group of sponsors participating in the summit and it’s worth a quick ‘guided tour’ of the teams that are helping us to pull off this amazing event.
+
+<a href="{{< relref "/cloud-engineering-summit" >}}" class="btn">Save your spot &#x2192;</a>
```

Comment:
```suggestion
<a href="{{< relref "/cloud-engineering-summit/?utm_campaign=ces-2021&utm_source=sponsor_blog&utm_medium=web" >}}" class="btn">Save your spot &#x2192;</a>
```

Reply:
```suggestion
<a href="{{< relref "/cloud-engineering-summit/?iad=sponsor-blog" >}}" class="btn">Save your spot &#x2192;</a>
```

Reply:
@zchase that's fine - we just want to measure the impact since we're hoping for partner amplification here.